_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+has a large collection of programmable boards. You will find boards like
+[Raspberry Pi](https://en.wikipedia.org/wiki/Raspberry_Pi), [Bana
+Pi](https://en.wikipedia.org/wiki/Banana_Pi),
+[Dragonboard](https://developer.qualcomm.com/hardware/dragonboard-410c),
+[Bigalbon](https://www.bigalsmods.com/product-p/1001.htm),
+[BBC-microbit](https://en.wikipedia.org/wiki/Micro_Bit) and the 3D printer. More
+on, this space has a great collection of books on Compilers and Embedded
+programming. I managed to found few on Open source too. The owners are great
+hackers. You will love to interact with hacker [Siddhesh
+Poyarekar](https://siddhesh.in/). Hacker [Nisha
+Poyarekar](https://twitter.com/nisha_poyarekar) is volunteering the [PyLadies
+community](https://www.meetup.com/PyLadies-Pune/) at Pune.
+
+
+#### Pune to Mumbai
+
+I spent my half day in this space. I got the responsibility of picking one of
+the keynote speakers landing at Mumbai airport midnight. To be frank, estimation
+of Google map between Pune to Mumbai is wrong. It showed nearly 2 hours but it
+took almost 4.5 hours to reach Mumbai. Few more minutes to reach the airport.
+The road is impressively smooth. You will encounter beautiful mountains of
+[Lonavla](https://en.wikipedia.org/wiki/Lonavla). The task of moving from Pune
+to Mumbai airport, receive Katie and come back to Pune completed in almost 13
+hours. I left from Pune around 4.30 PM and came back nearby 5 AM next day early
+morning.
+
+#### Illness during conference
+
+Because I did the huge amount of traveling at that night, I was unable to get
+enough sleep. Such tiredness resulted as an eye infection. I managed to attend
+the first day of the conference, but I was not in a condition to attend the
+second day. Treatment of local doctor healed me 2 days and then I was able to
+take part into Dev sprints.
+
+### Conference
+
+The conference was of total 4 days where initial two days were for the talks and
+end was assigned for a Devsprint. It didn't bombard with many tracks but gave
+quality talks presented in a single track. Timings were from 9 AM to 5 PM which
+was taken little lightly by the attendees.
+
+I was pretty impressed with the keynote speakers of this conference.
+
+#### [Katie Cunningham](https://twitter.com/kcunning)
+
+Katie is the O'Reilly author. Her
+[book](http://shop.oreilly.com/product/0636920024514.do) on Accessibility
+depicts her area of expertise. She is fun to talk. She likes to listen about
+developer communities, writing and most importantly computer games. Her broad
+vision on product development is amazing. She is an avid reader. I enjoyed
+listening to her experience of being in India very first time.
+
+#### [Honza Kral](https://twitter.com/honzakral)
+
+Honza is the dude who loves contributing to
+[Django](https://www.djangoproject.com/). He is the core contributor of Django
+too. He hacks on Python drivers at the [Elastic](https://www.elastic.co/). I was
```

Comment:
Typically a company name wouldn't have "the" in front, so "the Elastic" should probably just be "Elastic".